node-operators/networks/full-node.md

```diff
@@ -119,7 +118,6 @@ Now, execute the docker run command. Note that you have to:
     purestake/moonbeam:{{ networks.moonbase.parachain_docker_tag }} \
     --ws-external \
     --rpc-external \
```

Comment:
You're recommending these for MacOS but not for Ubuntu.

Reply:
Yes. My understanding is that network="host" doesn't work on mac so you have to provide these extra flags.

Reply:
good point, fixed.

Reply:
it was the --ws-external and --rpc external.  i left the host flags